app/furniture/marketplace/docs/README.md

```diff
@@ -0,0 +1,137 @@
+* [Developer tools](https://stripe.com/docs/development)
+* [Testing](https://stripe.com/docs/testing)
+
+
+### Webhooks
+Almost every application integrating with a payment provider will require some form of bi-directional communication, and an architectural pattern that can handle asynchronous actions or events that occur on the provider's side. Even at their most basic, payment workflows between two systems get complicated fast. Rather than forcing you to proactively poll their system for updates, Stripe supports custom webhooks in your own application to monitor and receive messages. Our Marketplace gizmo uses Stripe webhooks, so make sure to familiarize yourself with how events and webhooks work at Stripe.
+* [Use incoming webhooks to get real-time updates](https://stripe.com/docs/webhooks)
+* [Stripe webhook events overview](https://stripe.com/docs/webhooks/stripe-events)
+* [Test a webhooks integration with the Stripe CLI](https://stripe.com/docs/webhooks/test)
+
+### Setting up your local environment
+
+Since Marketplace already implements Stripe with custom webhook endpoints, developers will only need to setup the tooling to build and test against the existing code.
+
+#### Create a Stripe account and connect a Marketplace
+
+These instructions assume you don't already have a Stripe account. Developers who are familiar with Stripe and have developed with Stripe may skip steps as appropriate.
+
+First, let's create and configure a Stripe account for testing:
+1. [Sign up](https://dashboard.stripe.com/register)
+  - Consider using a different email from your personal email. Eg, `my.name+stripedev@gmail.com`.
+  - Since you will only be using Stripe to test, you won't have to pay anything
+1. When logged into your Stripe account, turn on `Test mode` in the top right of the dashboard.
+  ![Turn on Stripe test mode](stripe_test_mode.png)
+
+For the "Connect to Stripe" flow to work, you will need to have a local development URL that is recognized by Stripe as "reachable". One way to do this is:
+1. Add `127.0.0.1 convene.local` to your `/etc/hosts`
+1. Access the app locally through `convene.local:3000`, instead of `localhost`
+1. Optionally, add `convene.local` as the `branded_domain` to whatever Space you are using for testing, to shorten the URL you use to access it
+
+Next, add your secret API key:
+1. Click on the `Developers` button next to `Test mode` in the top right of your Stripe dasbhoard
+1. Find your `Secret key` and copy it
+1. Paste the key into `Api token` field on your Stripe Utility settings page
+  - http://convene.local:3000/spaces/<space_name>/utilities/<utiltiy_id>/edit
```

Comment:
is this supposed to render as a partial link to help readers quickly jump to the correct page? Or is it supposed to be quoted like so 👇 ?
```suggestion
  - `http://convene.local:3000/spaces/<space_name>/utilities/<utiltiy_id>/edit`
```